bridge/client/app/_pipes/truncate-number.pipe.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { TruncateNumberPipe } from './truncate-number';
+
+describe('TruncateNumberPipe', () => {
```

Comment:
Maybe also add a test for negative numbers and a case to truncate a number to 3 or 1 decimals (a parameter that isn't 2)